### PR TITLE
Sterling Wallet and Bill Payment

### PR DESCRIPTION
**Pull request by mezlet**
Add Bill payment
Add Wallet
Test modules

### CODE REVIEW

File: src/Sterling/README.md [hunk 1]
```diff
@@ -173,6 +173,297 @@ echo json_encode($instance5->InterbankTransferReq([
...
+               "Referenceid" => "01",
+               "RequestType" => "0",
+               "Translocation" => "01",
+               "amt" => "any:string",
+               "tellerid" => "any:string",
+               "frmacct" => "any:string",
+               "toacct" => "any:string",
+               "exp_code" => "any:string",
+               "paymentRef" => "any:string",
+               "remarks" => "any:string"
```
Comment (akinjide) [inline src/Sterling/README.md]: But random values here but add instruction telling users they can provide any string value

File: src/Sterling/README.md [hunk 2]
```diff
@@ -173,6 +173,297 @@ echo json_encode($instance5->InterbankTransferReq([
...
+            "Referenceid" => "01",
+            "RequestType" => "0",
+            "Translocation" => "01",
+            "amt" => "any: string",
+            "paymentcode" => "any: string",
+            "mobile" => "any: string",
+            "SubscriberInfo1" => "any: string",
+            "ActionType" => "any: string",
+            "nuban" => "any: string",
+            "email" => "any:string"
```
Comment (akinjide) [inline src/Sterling/README.md]: Same applies here too

File: src/Sterling/README.md [hunk 3]
```diff
@@ -173,6 +173,297 @@ echo json_encode($instance5->InterbankTransferReq([
...
```
Comment (akinjide) [inline src/Sterling/README.md]: Same applies here too

File: src/Sterling/README.md [hunk 4]
```diff
@@ -173,6 +173,297 @@ echo json_encode($instance5->InterbankTransferReq([
...
+
+echo json_encode($instance->BillersPaymentItems([
+        'sandbox_key' => 'Your sandbox_key',
+        'subscription_key' => 't',
+        'params'  => [
+            "Referenceid" => "01",
+            "RequestType" => "0",
+            "Translocation" => "01",
+            "Bvn" => "any: string",
+            "billerid" => "any: string"
```
Comment (akinjide) [inline src/Sterling/README.md]: Same applies here too

File: src/Sterling/Account.php [hunk 5]
```diff
@@ -20,8 +20,9 @@ public function InterbankTransferReq($data)
         try {
             $data['path'] = '/sterling/accountapi/api/Spay/InterbankTransferReq';
             $payload = isset($data['payload']) ? $data['payload'] : '';
+            $host = isset($data['host']) ? $data['host'] : '';
```
Comment (akinjide) [inline src/Sterling/Account.php:23]: What host are you setting to `""` here?

**Review by akinjide — CHANGES_REQUESTED** [review]
Minor changes

File: src/Sterling/README.md (hunk 1 above)
Comment (mezlet) [reply src/Sterling/README.md]: done

File: src/Sterling/README.md (hunk 3 above)
Comment (mezlet) [reply src/Sterling/README.md]: done

File: src/Sterling/README.md (hunk 4 above)
Comment (mezlet) [reply src/Sterling/README.md]: done

File: src/Sterling/Account.php (hunk 5 above)
Comment (mezlet) [reply src/Sterling/Account.php:23]: the host is not compulsory so I am setting it to '' if it not provided.

File: src/Sterling/README.md (hunk 2 above)
Comment (mezlet) [reply src/Sterling/README.md]: done

**Review by akinjide — APPROVED** [review]

**Pull request merged**